OneCommunityGlobal/HighestGoodNetworkApp
# Ashmita/lintissuefix

# Description
Fixed linting errors across multiple components. 

## Related PRS (if any):
This frontend PR is not related to any backend PR.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user
5. The App should work without breaking.
6. Run `npm run test` to make sure all tests are passing.
